google/cloud/bigtable/tests/admin_backup_integration_test.cc

```diff
@@ -23,6 +23,7 @@
 #include "google/cloud/testing_util/is_proto_equal.h"
 #include "google/cloud/testing_util/status_matchers.h"
 #include <gmock/gmock.h>
+#include <algorithm>
```

Comment:
Q. Is this a remnant from the previous edit, or something that needed to be added anyway?

Reply:
Remnant. Removed.